code/modules/client/preferences.dm

```diff
@@ -1032,7 +1033,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 		if(1)
 			jpval = JP_HIGH
 
-	if(role == SSjob.overflow_role)
+	if(job.type == SSjob.overflow_role)
```

Comment:
Here's some more type comparisons I spotted.

Reply:
Yes, but in this case it's justified, since the overflow role is a specific type.